kitty-items-cadence/contracts/Kibble.cdc

```diff
@@ -1,4 +1,4 @@
-import FungibleToken from 0xFUNGIBLETOKENADDRESS
+import FungibleToken from "./FungibleToken.cdc"
```

Comment:
I'd prefer if we could find a way to import contracts without having to copy and paste them to the same repo

Reply:
Yeah, agreed. I left my thoughts here, but I think we could make this work with URL imports: https://github.com/onflow/flow-cli/issues/71#issuecomment-783993348